cpp/.clang-tidy

```diff
@@ -39,8 +39,8 @@ Checks:
        -clang-analyzer-optin.core.EnumCastOutOfRange,
        -clang-analyzer-optin.cplusplus.UninitializedObject'
 
-WarningsAsErrors: '*'
-HeaderFilterRegex: '.*cudf/cpp/(src|include|tests).*'
+WarningsAsErrors: ''
```

Comment:
This change makes clang-tidy warnings not be errors so the CI build doesn't get marked as a failure and always runs to completion.